src/components/commandHandler.ts

```diff
@@ -138,7 +127,8 @@ export abstract class CommandHandler {
     }
 
     static IdentifyHandler(): void {
-        if (!this.playbackManager.isPlaying()) {
+        if (!PlaybackManager.isPlaying()) {
+            DocumentManager.setAppStatus('waiting');
```

Comment:
Can't we make these app status an enum? DocumentManager seems a good place to declare it.

Reply:
I agree

Reply:
Fixed in https://github.com/hawken93/jellyfin-chromecast/pull/228